packages/waku/src/lib/builder/serve-cloudflare.ts

```diff
@@ -8,7 +8,7 @@ const loadEntries = () => import(import.meta.env.WAKU_ENTRIES_FILE!);
 let serveWaku: ReturnType<typeof honoMiddleware> | undefined;
 
 const app = new Hono();
-app.use('*', serveStatic({ root: './' }));
+app.use('*', serveStatic({ root: './' } as any)); // FIXME we manifest should be optional
```

Comment:
@yusukebe Hi, `manifest` can be optional, right? (Seems so according to the source code.)

Reply:
resolved.